Linear_cell_complex/include/CGAL/Linear_cell_complex_constructors.h

```diff
@@ -339,9 +339,9 @@ namespace CGAL {
           else if ( it->vertex(1) == atr.infinite_vertex() )
             dart = alcc.beta(res, 1);
           else if ( it->vertex(2) == atr.infinite_vertex() )
-            dart = alcc.beta(res, 2);
+            dart = alcc.beta(res, 1, 1);
```

Comment:
You are right, beta(res, 2) and beta(res, 1, 1) do not describe the same vertex. In fact beta(res, 2) point to vertex(1).

And thus there is a bug here for vertex(2). 

Please could you remove the second modification (line 344) ?